doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,216 @@
+
+Submit run scripts - test recipe runs
+-------------------------------------
+
+We are now ready to start running all the available recipes, to compare output against previous release. Running is currently done
+via batch scripts submitted to a schedulers (SLURM). Generate the submission scripts using the ``generate.py`` `Python script
+<https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/utils/batch-jobs/generate.py>`_. 
+
+You will have to set the name of your environment, your email address (if you want to get email notifications for successful/failed jobs) and the name of the directory you want to store the log files of the jobs. A compute project from which resources are billed needs to be set, and the default partition is set to `compute`.
+More information on running jobs with SLURM on DKRZ/Levante can be found in the DKRZ `documentation
+<https://docs.dkrz.de/doc/levante/running-jobs/index.html>`_.
+
+Some recipes need other job requirements, you can add their headers in the `SPECIAL_RECIPES` dictionary. Otherwise the header will be written following the template that is written in the lines below. If you want to exclude recipes, you can do so by uncommenting the exclude lines.
+
+The launch scripts will be saved in the same directory you execute the script from. These are named like ``launch_recipe_<name>.sh``.
+To submit these scripts to the SLURM scheduler, use the ``sbatch launch_recipe_<name>.sh`` command. You can check the status of your BATCH queue by invoking:
+
+.. code-block:: bash
+
+  squeue -u $USER
+
+Also, for computationally-heavy recipes, you can require more memory and/or time, see e.g. edited batch header below
+(note the `compute` partition which is used for such heavy runs):
+
+.. code-block:: bash
+
+  #SBATCH --partition=compute
+  #SBATCH --time=08:00:00
+  #SBATCH --constraint=512G
+
+.. note::
+
+  On DKRZ/Levante, a user can't have more than 20 SLURM jobs running at a time.
+  As soon as a job is finished, the next one should start. More information on the job handling at DKRZ `here
+  <https://docs.dkrz.de/doc/levante/running-jobs/partitions-and-limits.html#levante-partitions-and-limits>`_.
+
+Analyse the results
+-------------------
+
+Once all jobs are completed, assemble some statistics so that issues with certain recipes
+can be followed-up, and document this information in the release issue, such as:
+
+- number of successfully run recipes
+- number of failed recipes with preprocessor errors (can they be fixed? Can the fixes be included in the release?)
+- number of failed recipes with diagnostic errors (can they be fixed? Can the fixes be included in the release?)
+- number of recipes that are missing data
+- number of recipes that have various other issues (and document them)
+
+To parse the output of all these runs, use the ``parse_recipes_output.py`` `utility Python script
+<https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/utils/batch-jobs/parse_recipes_output.py>`_.
```

Comment:
It would be nicer to link to the documentation describing this script instead of to the actual code